pkg/lib/localstore/ftsearch/ftsearchtantivy.go

```diff
@@ -345,7 +347,10 @@ func (f *ftSearchTantivy) DocCount() (uint64, error) {
 
 func (f *ftSearchTantivy) Close(ctx context.Context) error {
 	f.schema = nil
-	f.index.Free()
+	if !f.isRunning {
```

Comment:
feels like it can be more clear here to just check for index field itself? You already nilling the schema, so you may as well nil index at the end

Reply:
I thought about null object, works in this case. Fixed